meta-mbl-distro/recipes-security/ecryptfs-utils/ecryptfs-utils_%.bbappend

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+
+SRC_URI += "\
+    file://ecryptfs.service \
```

Comment:
I think we should go with the minimal approach to begin with. The new ecryptfs.service on changes the comment and runs ecryptfs-init.sh rather than ecryptfsd, which are simple changes.

Reply:
Track this issue with this jira ticket: https://jira.arm.com/browse/IOTMBL-2366